src/input.ts

```diff
@@ -246,3 +216,64 @@ export class Mouse {
     await this.click(x, y, { ...options, clickCount: 2 });
   }
 }
+
+const aliases = new Map<string, string[]>([
+  ['ShiftLeft', ['Shift']],
+  ['ControlLeft', ['Control']],
+  ['AltLeft', ['Alt']],
+  ['MetaLeft', ['Meta']],
+  ['Enter', ['\n', '\r']],
+]);
+
+const qwertyLayout = buildLayoutClosure(keyboardLayout.Qwerty);
+
+function buildLayoutClosure(layout: keyboardLayout.KeyboardLayout): Map<string, KeyDescription> {
+  const result = new Map<string, KeyDescription>();
+  for (const code in layout) {
+    const definition = layout[code];
+    const description: KeyDescription = {
+      key: definition.key || '',
+      keyCode: definition.keyCode || 0,
+      keyCodeWithoutLocation: definition.keyCodeWithoutLocation || definition.keyCode || 0,
+      code,
+      text: definition.text || '',
+      location: definition.location || 0,
+    };
+    if (definition.key.length === 1)
+      description.text = description.key;
+
+    // Generate shifted definition.
+    let shiftedDescription: KeyDescription | undefined;
+    if (definition.shiftKey) {
+      assert(definition.shiftKey.length === 1);
+      shiftedDescription = { ...description };
+      shiftedDescription.key = definition.shiftKey;
+      shiftedDescription.text = definition.shiftKey;
+      if (definition.shiftKeyCode)
+        shiftedDescription.keyCode = definition.shiftKeyCode;
+      description.shifted = shiftedDescription;
+    }
+
+    // Map from code: Digit3 -> { ... descrption, shifted }
+    result.set(code, { ...description, shifted: shiftedDescription });
```

Comment:
`description.shifted` is already set, so this is equivalent to result.set(code, description);
```suggestion
    result.set(code, description);
```

`shiftedDescription` can be const inside the if block as well then.

Reply:
It is actually backwards - i want no 'shifted' inside `3` or `#`, only inside `Digit3`.